internal/generator/backend/template/zkpschemes/plonk/plonk.setup.go.tmpl

```diff
@@ -89,7 +88,7 @@ type ProvingKey struct {
 }
 
 // TODO modify the signature to receive the SRS in Lagrange form (optional argument ?)
-func Setup(spr *cs.SparseR1CS, kzgSrs kzg.SRS) (*ProvingKey, *VerifyingKey, error) {
+func Setup(spr *cs.SparseR1CS, srs, srsLagrange kzg.SRS) (*ProvingKey, *VerifyingKey, error) {
```

Comment:
Maybe if `srsLagrange == nil` we compute it inside Setup?

The function documentation doesn't reflect anymore that also have to provide srs in Lagrange form or how to get it.